### PR TITLE
[ENDPOINT] PUT /member/:id - OT102-91

### PR DESCRIPTION
**Pull request by JonatanGarbuyo**
[OT102-91](https://alkemy-labs.atlassian.net/browse/OT102-91)
## Description:

AS user (admin)
I WANT to update Members
TO keep the information up to date.

## Acceptance criteria:
PUT / members /: id - You will need to validate that the member exists and update it,
otherwise return an error

### field validation:
val-member-err

#### members before 
members-before

#### members after
members-after

#### member not found error
not-found

### CODE REVIEW

File: routes/members.js [hunk 1]
```diff
@@ -11,5 +11,7 @@ const router = express.Router()
 router.get('/', get)
 /* POST new member */
 router.post('/', validateRequest(memberSchema), post)
+/* UPDATE new member */
+router.put('/:id', validateRequest(memberSchema), update)
```
Comment (dvelezroman) [inline routes/members.js]: falta el middleware para isAdmin

**Review by dvelezroman — CHANGES_REQUESTED** [review]

File: routes/members.js [hunk 2]
```diff
@@ -11,5 +12,7 @@ const router = express.Router()
 router.get('/', get)
 /* POST new member */
 router.post('/', validateRequest(memberSchema), post)
+/* UPDATE new member */
+router.put('/:id', [isAdmin, validateRequest(memberSchema)], update)
```
Comment (dvelezroman) [inline routes/members.js:16]: cool

**Review by dvelezroman — APPROVED** [review]
Very good PR

**Review by joselo85 — APPROVED** [review]

**Review by gabyMangoni — APPROVED** [review]
ok

**Review by alexiscjscab — APPROVED** [review]

**Pull request merged**